simplivity/resources/hosts.py

```diff
@@ -180,3 +180,16 @@ def get_virtual_controller_shutdown_status(self):
         """Retrieves the shutdown status of the Virtual Controller"""
         resource_uri = "{}/{}/virtual_controller_shutdown_status".format(URL, self.data["id"])
         return self._client.do_get(resource_uri)
+
+    def cancel_virtual_controller_shutdown(self, timeout=-1):
+        """Cancels the virtual controller shutdown.
+
+        Args:
+          timeout: Time out for the request in seconds.
+
+        Returns:
+          object: Response object.
```

Comment:
I don't think this is returning an object, isn't it just a `dict` ?

Reply:
@sijeesh should we just return the status string?

Possible values are "STRING ['SUCCESS', 'FAILURE', 'UNKNOWN', 'IN_PROGRESS' ]

Reply:
Updated as per suggestion returning the string response now

Reply:
@svt-kmcdaniel , Yes, a status string should be fine here.